leakcanary-android-core/src/main/java/leakcanary/internal/NotificationReceiver.kt

```diff
@@ -39,7 +40,12 @@ internal class NotificationReceiver : BroadcastReceiver() {
     ): PendingIntent {
       val broadcastIntent = Intent(context, NotificationReceiver::class.java)
       broadcastIntent.action = action.name
-      return PendingIntent.getBroadcast(context, 0, broadcastIntent, 0)
+      val flags = if (Build.VERSION.SDK_INT > 30) {
```

Comment:
Using `Build.VERSION_CODES.R` requires updating `compileSDK` from 29 to 30. This update caused another issues. So, it is better to do `compileSDK` update in separate PR